packages/schematics/angular/app-shell/index_spec.ts

```diff
@@ -10,6 +10,21 @@ import { Schema as ApplicationOptions } from '../application/schema';
 import { Schema as WorkspaceOptions } from '../workspace/schema';
 import { Schema as AppShellOptions } from './schema';
 
+function createServerFile(appTree: UnitTestTree): void {
```

Comment:
This is not needed.

Reply:
Ohh got it, So my impression was incorrect, I will make the changes.